### DISCUSSION TITLE
Apparently node-sass is deprecated. What is the replacement?

### DISCUSSION THREAD

**Discussion by djpirra** [silevis/reactgrid#242] [General]
Hi everyone.

How are you guys defining your grids styling?
I am new to reactgrid and was just trying to install node-sass as per the documentation when I realized it is deprecated.

Additionally it appears to give an error installing on my alpine docker image due to Python not being installed.... really?

Would appreciate any assistance.

Thank you all.

**Comment by djpirra** [comment]
Any comments on this?

**Comment by Michota** [comment]
If this is still a problem, [create issue from discussion](https://github.com/silevis/reactgrid/issues/new?created_from_discussion_number=242).
